src/fetchStats.js

```diff
@@ -1,93 +0,0 @@
-const AWS = require('aws-sdk')
-
-const awsConfig = {
```

Comment:
Typically you will access data via some endpoint. Not a problem at all considering this is just a small project. It's more work to stand up a single endpoint for data lol.

Reply:
Would y'all use something like API Gateway for this?

Reply:
On the FE we'll never connect to any DB directly. We'll always access the data through some backend API that we setup. I believe API gateway stands in between the backend and user for rate limiting and other stuff (idk enough about API gateway but I heard it does a lot more)

e.g. api.something.com/emails => returns emails for a user